packages/loader/container-loader/src/connectionManager.ts

```diff
@@ -755,6 +759,9 @@ export class ConnectionManager implements IConnectionManager {
 		connection.off("disconnect", this.disconnectHandlerInternal);
 		connection.off("error", this.errorHandler);
 		connection.off("pong", this.props.pongHandler);
+		if (connection.supportDeltaConnectionUpdates === true) {
```

Comment:
can we always subscribe? I assume no, as it will throw if Driver is on different version.